client/src/components/ReviewTile.js

```diff
@@ -0,0 +1,24 @@
+// prepare formatted review tiles 
+import React, { useEffect, useState } from "react";
+
+const ReviewTile = ({ review }) => {
+  let body, username, createdAtString, score
+  
+  if(review) {
+    body = <p>{review.body}</p>
+    username = <p>{review.username}</p>
+    score = <p>{review.score}</p>
+    createdAtString = <p>{(new Date(review.createdAt)).toUTCString()}</p>
+  }
+
+  return (
+    <div className="callout">
+      <h5>User:{username}</h5>
+      <h5>Created At: {createdAtString}</h5>
+      <h5>Score: {score}</h5>
+      <h5>Review: {body}</h5>
+    </div>
+    )
+}
```

Comment:
the parentheses on line 21 should line up with the indentation on line 14 ("outdent" by one)